src/clarity-angular/data/datagrid/datagrid.ts

```diff
@@ -68,7 +68,7 @@ export class Datagrid implements AfterContentInit, AfterViewInit, OnDestroy {
     /**
      * Output emitted whenever the data needs to be refreshed, based on user action or external ones
      */
-    @Output("clrDgRefresh") public refresh = new EventEmitter<State>(false);
+    @Output("clrDgRefresh") public refresh = new EventEmitter<State>(true);
```

Comment:
You can revert this now that the proper fix is in.

Reply:
This has been done.